src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -530,19 +541,116 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     }
   }, [poolAsset, inboundAddressesData?.address])
 
-  // We reuse lending utils here since all this does is estimating fees for a given deposit amount with a memo
-  // It's not going to be 100% accurate for EVM chains as it doesn't calculate the cost of depositWithExpiry, but rather a simple send,
-  // however that's fine for now until accurate fees estimation is implemented
+  const thorchainNotationPoolAssetId = useMemo(() => {
+    if (!poolAsset) return undefined
+    return assetIdToPoolAssetId({
+      assetId: poolAsset.assetId,
+    })
+  }, [poolAsset])
+
+  const memo = useMemo(() => {
+    if (!runeAccountId) return undefined
+    if (thorchainNotationPoolAssetId === undefined) return undefined
+    // Note, bps is a placeholder and not the actual bps here, this memo is just used to estimate fees
+    return `+:${thorchainNotationPoolAssetId}:${fromAccountId(runeAccountId).account ?? ''}:ss:50`
+  }, [runeAccountId, thorchainNotationPoolAssetId])
+
+  const estimateFeesArgs = useMemo(() => {
+    if (
+      !assetId ||
+      !wallet ||
+      !poolAsset ||
+      !poolAsset ||
+      !memo ||
+      !feeAsset ||
```

Comment:
we aren't using `feeAsset`

Reply:
d2428a405fc4d0ab4a7534bcf545dd150ff1e13c